src/main/kotlin/com/sourcegraph/cody/context/ui/EnhancedContextPanel.kt

```diff
@@ -80,6 +80,10 @@ constructor(protected val project: Project, protected val chatSession: ChatSessi
 
   /** Gets the chat session's enhanced context state. */
   protected fun getContextState(): EnhancedContextState? {
+    if (CodyAuthenticationManager.getInstance(project).getActiveAccount() == null) {
```

Comment:
Why is that addition needed? Is that a real scenario which happen during your testing?
One could argue that even if there is no active account there is always default context, so that comment is not really true.